reference/public-ip-addresses.md

```diff
@@ -18,3 +20,23 @@ North American region (us.platform.sh):
 These addresses are stable, but not guaranteed to never change. Prior to
 any future change, all affected customers will receive ample warning.
 
+## Inbound IP addresses
+
+Platform.sh regions accept connections on a limited number of IP
+addresses. You may need to whitelist them on your corporate firewall
+to enable SSH connections.
```

Comment:
This doesn't make sense. We accept inbound connections from any IP at all.  Even incoming SSH I can connect from an arbitrary IP, as long as I have keys setup.  What are we talking about here?